src/features/settings/screens/UsersScreen/UsersScreen.tsx

```diff
@@ -41,33 +53,47 @@ const UsersScreen = () => {
     console.log(user);
   };
 
+  const handleDuplicate = (user: UserDef) => {
+    // TODO: API to duplicate user
+    console.log(user);
+  };
+
   const handleEdit = (user: UserDef) => {
     history.push(
       generatePath(SettingsPathsEnum.USERS_EDIT, { id: user.id.toString() })
     );
   };
   const handleCloseModal = () => {
     history.push(SettingsPathsEnum.USERS);
-    setShowEditModal(false);
+    setShowUserModal(false);
+  };
+
+  const handleAddUserModal = () => {
+    history.push(generatePath(SettingsPathsEnum.USERS_CREATE));
   };
 
   return (
     <>
-      <h1>{t("settingsUsers.title")}</h1>
-      <p>{t("settingsUsers.text")}</p>
+      <ScreenTitleView mainTitle={t("settingsUsers.title")} />
+      <div className={styles.buttonContainer}>
+        <Button type="primary" size="large" onClick={handleAddUserModal}>
```

Comment:
you can actually just use the `to` props instead of onClick 😄 
```suggestion
        <Button type="primary" size="large" to={SettingsPathsEnum.USERS_CREATE}>
```